app/views/movement_confirmation_page.scala.html

```diff
@@ -16,50 +16,58 @@
 
 @import forms.{Choice, ConsignmentReferences}
 @import views.Title
+@import forms.MucrOptions
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import views.Title
+@import components.gds.{errorSummary, sectionHeader, gds_main_template}
+@import components.gds.exportsInputText
```

Comment:
I don't see this import being used anywhere.

Reply:
Thanks.   Organise imports does seem to work.